packages/kotti-ui/source/next/kotti-field/KtField.vue

```diff
@@ -282,20 +297,27 @@ export default defineComponent({
 
 			&__tooltip {
 				position: absolute;
-				bottom: 20px;
-				display: none;
-				padding: 0.25em;
-				color: var(--support-info);
+				z-index: 1; // fix weird layout issues with el-select
+				width: max-content;
+				max-width: 12rem;
+				max-height: 7em;
+				padding: 0.75em 1em;
+				overflow: scroll;
+				color: var(--gray-10);
```

Comment:
Can't we use a token here?

Reply:
@tomquinonero which token are you suggesting? Personally, I never know which token belongs to what token as the names seem rather arbitrary to me.

Personally, I’d prefer it if we had something like: `ui-01-background` + `ui-01-foreground` so that it’s clear that they should be used together. With the current tokens, it feels like `gray-10` with extra steps to me but perhaps I just don’t get it